docs/blockchain/contracts/nft-contract/nft-actions/stofrcfg.a.experimental.md

```diff
@@ -0,0 +1,70 @@
+---
+title: 'stofrcfg.a'
+order: 36
+
+---
+
+# stofrcfg.a
+
+Set global uniq offer configuration
+
+## Behavior
+
+Set global configuration for all uniq offer include both token and token factory.
+
+## Technical Behavior
+
+Only `ultra.nft.ft` account can set buy offer config
+
+The action stores the config in to `offercfg.a` singleton with the specified arguments.
+
+All fields are optional, it will only set the specified arguments and leave the rest the same as existing entry or as the default value if there’s no existing entry.
+
+This action allows one to set the same config and won’t assert.
+
+`eosio.nft.ft` pays RAM usage.
+
+`min_price` must be positive, and only support UOS and USD.
+
+`min_duration` and max_duration must also be positive, with max_duration > min_duration.
```

Comment:
Should be
Both `min_duration` and `max_duration` must be positive, and `max_duration` must be greater than `min_duration`.

Reply:
updated